server/src/firestore-api.ts

```diff
@@ -112,14 +113,18 @@ export const resetMuxFirestoreRelationship = async (roomID: string) => {
 /**  */
 export async function managePresenceInDB(allRoomNames: string[]) {
   const presenceRef = firestore.collection("presence");
-  
   const currentlyOnline = await Promise.all(allRoomNames.map(async (streamName) => {
-    let q = presenceRef.where("room_id", "==", streamName).where("timestamp", ">=", Timestamp.fromMillis(Date.now() - (1.2 * PRESENCE_LENGTH)));
+    //let q = presenceRef.where("room_id", "==", streamName).where("timestamp", ">=", Timestamp.fromMillis(Date.now() - (1.2 * PRESENCE_LENGTH)));
```

Comment:
maybe we can remove these rogue commented snippets before merging 👍

Reply:
Done!